ee/app/engagement-dashboard/client/components/UsersTab/ActiveUsersSection.js

```diff
@@ -12,14 +12,13 @@ import { downloadCsvAs } from '../../../../../../client/lib/download';
 
 const ActiveUsersSection = () => {
 	const t = useTranslation();
-
 	const period = useMemo(() => ({
 		start: moment().set({ hour: 0, minute: 0, second: 0, millisecond: 0 }).subtract(30, 'days'),
-		end: moment().set({ hour: 0, minute: 0, second: 0, millisecond: 0 }).subtract(1),
+		end: moment().set({ hour: 0, minute: 0, second: 0, millisecond: 0 }).subtract(1, 'days'),
 	}), []);
 
 	const params = useMemo(() => ({
-		start: period.start.clone().subtract(30, 'days').toISOString(),
+		start: period.start.clone().subtract(29, 'days').toISOString(),
```

Comment:
i'm curious, why was this needed? 👀 february?

Reply:
@KevLehman To compute the MAU for an arbitrary `x` date we need to sum over a **closed** time interval of `[x - 29, x]`, which give us 30 days.

Reply:
Ah, that makes sense 👀 thanks for sharing that